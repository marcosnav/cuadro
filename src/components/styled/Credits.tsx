import styled from 'styled-components';

const Credits = styled.p`
  bottom: 0;
  color: ${(props) => props.theme.TEXT};
  font-size: 0.7em;
  left: 0;
  line-height: 1.44em;
  position: fixed;
  text-align: center;
  width: 100%;
`;

Credits.displayName = 'Credits';

export default Credits;
