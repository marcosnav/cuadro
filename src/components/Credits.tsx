import React, { FC } from 'react';
import PrettyLink from './PrettyLink';
import * as S from './styled';

const Credits: FC = () => {
  return (
    <S.Text bottom={true}>
      {'Made with </> by '}
      <PrettyLink href='https://github.com/marcosnav/cuadro'>
        marcosnav.
      </PrettyLink>
      {' See project in '}
      <PrettyLink href='https://github.com/marcosnav/cuadro'>
        GitHub.
      </PrettyLink>
    </S.Text>
  );
};

export default Credits;
