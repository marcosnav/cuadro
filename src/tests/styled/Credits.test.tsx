import { mount } from 'enzyme';
import 'jest-styled-components';
import React from 'react';
import Credits from './../../components/styled/Credits';

describe('<Credits/>', () => {
  it('renders properly', () => {
    const component = mount(<Credits>Made by john</Credits>);
    expect(component.text()).toMatch('Made by john');
    expect(component).toMatchSnapshot();
  });
});
