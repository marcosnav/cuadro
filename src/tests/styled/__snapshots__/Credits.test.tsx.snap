// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Credits/> renders properly 1`] = `
.c0 {
  bottom: 0;
  font-size: 0.7em;
  left: 0;
  line-height: 1.44em;
  position: fixed;
  text-align: center;
  width: 100%;
}

<Credits>
  <p
    className="c0"
  >
    Made by john
  </p>
</Credits>
`;
